pontoon/machinery/views.py

```diff
@@ -90,16 +100,26 @@ def microsoft_translator(request):
         text = request.GET['text']
         locale_code = request.GET['locale']
     except MultiValueDictKeyError as e:
-        return HttpResponseBadRequest('Bad Request: {error}'.format(error=e))
+        return JsonResponse({
+            'status': False,
+            'message': 'Bad Request: {error}'.format(error=e),
+        }, status=400)
 
     api_key = settings.MICROSOFT_TRANSLATOR_API_KEY
 
     if not api_key:
         log.error("MICROSOFT_TRANSLATOR_API_KEY not set")
-        return HttpResponseBadRequest("Missing api key.")
+        return JsonResponse({
+            'status': False,
+            'message': 'Bad Request: {error}'.format(error='Missing api key.'),
```

Comment:
Hehe, I guess this can be simplified to a simple string. :wink:

Reply:
Yikes!